osmosis-1/osmosis.zone_assets.json

```diff
@@ -4158,6 +4158,26 @@
       "categories": ["liquid_staking"],
       "listing_date_time_utc": "2024-07-04T12:00:00Z",
       "_comment": "Stride Staked ISLM $stISLM"
+    },
+    {
+      "chain_name": "dymension",
+      "base_denom": "ibc/5A26C8DC8DA66F4DD94326E67F94510188F5F7AFE2DB3933A0C823670E56EABF",
```

Comment:
oh, at the moment this asset needs to be defined at the Chain Registry as well

Reply:
created a PR for the same - https://github.com/cosmos/chain-registry/pull/4957